Fixing Bug: Display github account name #8

Long315 opened the pull request:
* Fixing Bug: Display github account name #8

On the main screen, the gitHubName will be displayed on the right top of the screen, just next to 'log out' button.

Long315 committed:
Fixing Bug: Display github account name #8


* Fixing Bug: Display github account name #8

On the main screen, the gitHubName will be displayed on the right top of the screen, just next to 'log out' button.

Long315 committed:
Merge remote-tracking branch 'upstream/master'

Merge repository to my fork

sguy016 reviewed (changes requested):
The application can not be lunch anymore, we have a blank screen

Long315 committed:
delete wrong code caused by merging (#2)

* Testing Commit

* Fixing Bug: Display github account name #8

On the main screen, the gitHubName will be displayed on the right top of the screen, just next to 'log out' button.

* Revert "Testing Commit"

This reverts commit 1c0fa16fdaadaa5c4831fb0da2ea36ecfef5404c.

* Delete metaData in code

There is metaData in code which must be merged by mistake.
